ui/apps/platform/src/services/ComplianceEnhancedService.ts

```diff
@@ -1,10 +1,57 @@
+import axios from 'services/instance';
 import qs from 'qs';
 
 import { SearchFilter, ApiSortOption } from 'types/search';
 import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
 import { mockComplianceScanResultsOverview } from 'Containers/ComplianceEnhanced/Status/MockData/complianceScanResultsOverview';
 import { CancellableRequest, makeCancellableAxiosRequest } from './cancellationUtils';
 
+const scanScheduleUrl = '/v2/compliance/scan/configurations';
+
+type Schedule = UnsetSchedule | DailySchedule | WeeklySchedule | MonthlySchedule;
+
+type ScheduleIntervalType = 'UNSET' | 'DAILY' | 'WEEKLY' | 'MONTHLY';
+
+type UnsetSchedule = {
+    intervalType: 'UNSET';
+} & BaseSchedule;
+
+type DailySchedule = {
+    intervalType: 'DAILY';
+} & BaseSchedule;
+
+type WeeklySchedule = {
+    intervalType: 'WEEKLY';
+    // Sunday = 0, Monday = 1, .... Saturday =  6
+    daysOfWeek: {
+        day: number[]; // int32
+    };
+} & BaseSchedule;
+
+type MonthlySchedule = {
+    intervalType: 'WEEKLY';
+    // Sunday = 0, Monday = 1, .... Saturday =  6
+    daysOfMonth: {
+        day: number[]; // int32
+    };
```

Comment:
Aha, for the record, vulnerability reports 2.0 differs because no `'DAILY'` interval type:
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/report_service.proto#L62-L86